lib/tests/formulae.rb

```diff
@@ -586,6 +586,16 @@ def formula!(formula_name, args:)
           return
         end
 
+        if OS.linux? &&
+           tap.full_name == "Homebrew/homebrew-core" &&
+           ENV["HOMEBREW_REQUIRE_BOTTLED_LINUX"] &&
```

Comment:
This is the equivalent of the ARM flag, set here: https://github.com/Homebrew/homebrew-core/blob/master/.github/workflows/tests.yml#L101

I think it will allow to force a PR to build a Linux bottle even if it didn't have a Linux bottle before. We'll have to write the same code based than for ARM. But we might not need this and I can also remove the flag for the time being, if it's not used.

Reply:
> I think it will allow to force a PR to build a Linux bottle even if it didn't have a Linux bottle before.

Gotcha.